test/cpp/jit/test_gpu_view.cpp

```diff
@@ -922,6 +922,45 @@ TEST_F(NVFuserTest, FusionFlattenAfterUnsqueezeOutput_CUDA) {
   testValidate(&fusion, outputs, aten_inputs, {at_x_view}, __LINE__, __FILE__);
 }
 
+TEST_F(NVFuserTest, FusionComputeAtRootDomainMapWithView_CUDA) {
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  const std::vector<int64_t> input_shape1{10, 12};
+  const std::vector<int64_t> input_shape2{10, 3, 4};
+
+  auto tv0 = makeSymbolicTensor(2);
+  fusion.addInput(tv0);
+
+  auto tv1 = add(tv0, IrBuilder::create<Double>(1));
+
+  // reduction followed by broadcast
+  auto tv2 = sum(tv1, {1});
+  auto tv3 = broadcast(tv2, {false, true, true});
+
+  // Path with a view
+  auto tv4 = view(tv1, input_shape1, input_shape2);
+
+  // Join the reduciton+broadcast and view paths together
+  auto tv5 = add(tv3, tv4);
+  fusion.addOutput(tv5);
+
+  fusion.printMath();
```

Comment:
Is this a debugging print you forget to remove?

Reply:
My bad, sorry, I'll remove it.